Doc/library/inspect.rst

```diff
@@ -162,6 +162,12 @@ attributes (see :ref:`import-mod-attrs` for module attributes):
 |                 |                   | per-opcode events are     |
 |                 |                   | requested                 |
 +-----------------+-------------------+---------------------------+
+|                 | f_generator       | returns the generator or  |
```

Comment:
I'd list this between `f_locals` and `f_trace` because then it's grouped with `runtime info` rather than `tooling stuff`.